torchtune/utils/__init__.py

```diff
@@ -5,23 +5,6 @@
 # LICENSE file in the root directory of this source tree.
 
 from ._device import get_device
-from ._distributed import (  # noqa
-    contains_fsdp,
-    FSDPPolicyType,
```

Comment:
some of the removed imports are still in __all__

Reply:
ugh, too many merge conflicts